huaweicloud/data_source_enterprise_project.go

```diff
@@ -0,0 +1,84 @@
+package huaweicloud
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk/openstack/eps/v1/enterpriseprojects"
+)
+
+func dataSourceEnterpriseProject() *schema.Resource {
```

Comment:
use DataSourceEnterpriseProject to make it exportable.

Reply:
fixed